api/Services/ILabeledDataService.cs

```diff
@@ -0,0 +1,11 @@
+using api.Model;
+
+namespace api.Services;
+
+public interface ILabeledDataService
+{
+    Task<IList<LabeledData>> Get(int dataSetId);
+    Task<LabeledData> Create(LabeledData labeledData);
+    Task<LabeledData> Update(LabeledData labeledData);
+    Task Delete(int id);
```

Comment:
Delete hoort ook de verwijderde LabeledData te retourneren.
Misschien is een GetById op LabeledData ook chill

Reply:
delete return object heb ik een aparte taak voor aangemaakt, want dit is nu nergens zo
GetById heb ik toegevoegd voor LabeledData